src/account-type-utils.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Returns the account type based on the user's email address
+ * to use as a Mixpanel property on the user profile.
+ *
+ */
+
+const generalTld = [
+  'com', 'net', 'io', 'ai', 'co', 'app', 'us', 'uk', 'ca',
+  'de', 'fr', 'jp', 'au', 'ru', 'ch', 'se', 'no', 'nl', 'it',
+  'es', 'dk', 'cz', 'br', 'be', 'at', 'ar', 'in', 'mx', 'pl',
+  'pt', 'fi', 'gr', 'hk', 'id', 'ie', 'il', 'is', 'kr', 'my',
+  'nz', 'ph', 'sg', 'th', 'tw', 'vn', 'cn'
+]
+
+const domainTlds = {
+  'broadinstitute.org': 'Broad Employee',
+  'firecloud.org': 'Broad Employee',
+  'verily.com': 'Verily Employee',
+  'google.com': 'Verily Employee',
+  'gmail.com': 'Independent User',
+  'gserviceaccount.com': 'Service Account User',
+  'edu': 'Educational Institute User',
+  'org': 'Non-profit User',
+  'gov': 'Government User'
```

Comment:
Recommend adding `.` in front of each of these so we don't have to worry about it in code and to avoid incorrectly recognizing something like `myverily.com`.